io.github.yuxshao.ptcollab.yml

```diff
@@ -0,0 +1,31 @@
+id: io.github.yuxshao.ptcollab
+runtime: org.kde.Platform
+runtime-version: '5.15-23.08'
+sdk: org.kde.Sdk
+command: ptcollab
+modules:
+  - name: ptcollab
+    buildsystem: qmake
+    sources:
+      - type: git
+        url: https://github.com/yuxshao/ptcollab.git
+        tag: "v0.6.4.8"
+        commit: "5feb27decb9e0600d1defdaf9dc1424c6fdcb7ad"
+  - name: share
+    buildsystem: simple
+    sources:
+      - type: dir
```

Comment:
1. don't use `dir` source
2. only the SVG icon is necessary. remove all the others
3. no need for a separate module

Reply:
actually the icons are upstream they don't need to be here at all.

Reply:
Ok

Reply:
I have a problem with this, the original project doesn't has a metainfo file, I need somewhere to store it

Reply:
I didn't mention that one, so what's the problem?

Reply:
albeit it should be submitted for inclusion upstream for a future release (including installing it), as it's not flatpak specific.